elastic/kibana
# [Security Solution] Expandable flyout - Update index field in analyzer preview

## Summary

This PR addresses a bug that when a rule is created using data view, analyzer preview is blank. This is due to a check on if index exists before rendering the analyzer preview. This PR updated the index field from `kibana.alert.rule.parameter.index` to 'kibana.alert.rule.indices`, the later is introduced in https://github.com/elastic/kibana/pull/130929 and is available when a rule is created using either index patterns or data view.

**How to reproduce the bug**
- Refer to bug report https://github.com/elastic/kibana/issues/164829

**How to test**
- Create a rule using data view
- Generate some alerts
- Go to alerts page, expand a row in alerts table
- Go to Visualization -> Analyzer preview, the analyzer preview tree should be present

![image](https://github.com/elastic/kibana/assets/18648970/cbe0668e-335d-436a-992c-8970e75a3635)


### Checklist
